src/formats/CML.cpp

```diff
@@ -0,0 +1,581 @@
+// Chemfiles, a modern library for chemistry file reading and writing
+// Copyright (C) Guillaume Fraux and contributors -- BSD license
+
+#include <cassert>
+#include <sstream>
+
+#include <fmt/format.h>
+#include <fmt/ostream.h>
+
+#include "pugixml.cpp"
```

Comment:
Could we create a CMake OBJECT library instead ? This way if we implement other formats wich need XML, the library is already there